pkg/accounting/accounting.go

```diff
@@ -51,6 +54,7 @@ type accountingPeer struct {
 	lock             sync.Mutex // lock to be held during any accounting action for this peer
 	reservedBalance  uint64     // amount currently reserved for active peer interaction
 	paymentThreshold uint64     // the threshold at which the peer expects us to pay
+	surplus          bool       // signals wether peer has surplus to be used for debiting
```

Comment:
to know at debit whether we have any surplus balance to deduct from.
otherwise to check whether surplus balance is available, we'd do a store get for the surplus balance stored (in each debit). so this way we just check this bool, lighter